config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/CircuitProgrammi-AAAAAAAAAAAAAAAAAAAH_w==.json

```diff
@@ -31,7 +31,7 @@
+      "desc:8": "Some machines have a very, very large number of recipes. To avoid conflicts, many recipes have programmed circuits to ensure that a correct recipe is used. These programmed circuits are not consumed.\n\nIn this modpack, all singleblock GT machines now come with an icon in GUI on the bottom-right that represents a ghost circuit. Click it or scroll to choose the number you need. If you do that, you don\u0027t have to insert a physical circuit. This is going to be your main option for all single block machines. This option is now also available for input buses of multiblocks, which you will have in later tiers.\n\nThere are a few places throughout the modpack where you still need the physical circuits. You can turn an LV circuit into a programmed circuit in the crafting grid and then program it in various ways:\n\n1) Find the correct crafting recipe with a screwdriver in NEI and use that.\n\n2) Rightclick while holding the programmed circuit to open an easy menu. You still need to at least have the screwdriver in your inventory though or it won\u0027t work. Eventually you can also make an electric screwdriver.\n\n3) Use the BartWorks Circuit Programmer. It\u0027s only available in MV, but it does not use any durability and runs on EU.\n\n4) Use the GT++ Circuit Programmer block. It isn\u0027t portable and only available in MV, but it allows for automation.\n\n\nThe tasks in this quest are just the showcase of what programming curciuts there are. You can just click the checkmark once you understand how the programmed circuits work."
```

Comment:
"the correct recipe" not "a correct recipe". Or just leave as it was before, that was good.

"get to" was better then "have" but it's not really correct anyway. maybe "which you will use a lot later."

Reply:
Thank you! Fixed.